understanding/20/name-role-value.html

```diff
@@ -49,9 +49,14 @@ <h2>Intent of Name, Role, Value</h2>
       <p>A particularly important state of a user interface control is whether or not it has
          focus. The focus state of a control can be programmatically determined, and notifications
          about change of focus are sent to user agents and assistive technology. Other examples
-         of user interface control state are whether or not a checkbox or radio button has
-         been selected, or whether or not a collapsible tree or list node is expanded or collapsed.
+         of user interface control states are whether or not a checkbox or radio button has
+         been selected, or whether a collapsible tree view or disclosure widget is expanded or collapsed.
       </p>
+
+     <div class="note">
+        <p>While whether a control has focus or not is an important state that can be programmatically exposed to users,  
+        specific focus behaviors and keyboard accessibility are covered by other success criteria.</p>
+     </div>
```

Comment:
```suggestion
```

Reply:
@mbgower i can't think of a better way to word this, so going to go back on what i said during the meeting and follow your suggestion to remove it.